ofrak_type/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+All notable changes to `ofrak-type` will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+
+## [2.0.0](https://github.com/redballoonsecurity/ofrak/releases/tag/ofrak-type-v2.0.0) - 2023-01-03
+### Added
+- Add `ArchInfo` to describe an architecture.
+- Classes in package all discoverable at `ofrak_type`, e.g., `from ofrak_type import Range`.
+
+### Changed
+- Removed reference to obsolete `ofrak_components` from README.md.
+
+### Fixed
+- Fixed bug that flipped integer values of  `MemoryPermissions.R` and `MemoryPermission.X`.
```

Comment:
I consider this a "Change" - it's the reason why the major version changed.

Reply:
Ok -- I added it in `Change` as well.